tests/Orckestra.Composer.Cart.Tests/Factory/Order/OrderDetailsViewModelFactory_CreateViewModel.cs

```diff
@@ -223,5 +196,82 @@ protected Overture.ServiceModel.Orders.Order CreateOrder(string orderStatus, str
 
         }
 
+        [Test]
+        [TestCase("New|Pending", new string[] { "New" }, true)]
+        [TestCase("New|Pending", new string[] { "Pending" }, true)]
+        [TestCase("New|Pending", new string[] { "New", "Pending" }, true)]
+        [TestCase("New|Pending", new string[] { "Canceled" }, false)]
+        [TestCase("New|Pending", new string[] { "New,Canceled" }, false)]
+        public void WHEN_valid_request_EditableShipmentStates_SHOULD_return_TRUE(string editableShipmentStates,
```

Comment:
please rename to more correct name, for example

WHEN_order_with_provided_shipment_statuses_SHOULD_return_correct_IsOrderEditable